apps/oxfmt/src/format.rs

```diff
@@ -156,6 +175,50 @@ impl FormatRunner {
     }
 }
 
+const DEFAULT_CONFIG_NAMES: &[&str] = &[".oxfmtrc.json", ".oxfmtrc.jsonc"];
```

Comment:
@Sysix should we stick with just one config name? `.oxfmtrc.jsonc` ?

Reply:
Thank you both. I'll align it (only search `.json`) for now.